server/src/main/java/org/apache/iotdb/db/metadata/template/Template.java

```diff
@@ -339,7 +341,7 @@ private IMeasurementSchema constructSchema(
           for (IMNode child : cur.getChildren().values()) stack.push(child);
         }
       }
-    } catch (IllegalPathException e) {
+    } catch (MetadataException e) {
       e.printStackTrace();
     }
```

Comment:
Aovid e.printStackTrace. Use log

Reply:
get it, it will throw a exception rather than catch now.